### PR TITLE
4032 - UI: User list export with invitations

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

File: src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { useHeaders } from './hooks/useHeaders'
+
+const FILENAME = 'FRA-Users.csv'
+
+const Export: React.FC = () => {
+  const csvHeaders = useHeaders()
+  const data = useData()
+
+  return (
+    <CSVLink
```
Comment (minotogna) [inline src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:16]: Do we have a generic csv link component ?

Comment (sorja) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:16]: Do you mean the one in datatables?
Those export the table data, but in our case we export custom data from two tables

Let me know

Comment (minotogna) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:16]: I meant just for the ui style .
A wrapper to CsvLink imported .

Comment (minotogna) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:16]: If we don't , we'll think later on

Comment (sorja) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:16]: Let's add a note for this, I did not find this

https://github.com/orgs/openforis/projects/8?pane=issue&itemId=85627575

File: src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { useData } from './hooks/useData'
+import { useHeaders } from './hooks/useHeaders'
+
+const FILENAME = 'FRA-Users.csv'
+
+const Export: React.FC = () => {
+  const csvHeaders = useHeaders()
+  const data = useData()
+
+  return (
```
Comment (minotogna) [inline src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:15]: Wait . We'll need to use table paginated csv export generically.
We don't need it now this export

Comment (sorja) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:15]: We have two tables, should we have two buttons?

Comment (minotogna) [reply src/client/pages/CountryHome/Collaborators/UserList/Export/Export.tsx:15]: we'll do the export later, using generic db export. I would close this pr now

**Pull request closed without merging**